panda/src/pnmimagetypes/pnmFileTypeTIFF.cxx

```diff
@@ -1215,7 +1215,7 @@ void PNMFileTypeTIFF::
 tiff_warning(const char *, const char *format, va_list ap) {
   static const int buffer_size = 1024;
   char buffer[buffer_size];
-#if defined(WIN32_VC) || defined(WIN64_VC)
+#ifdef _WIN32
```

Comment:
Should probably be a `_MSC_VER` like in `pnmimage_base.cxx` (although it seems like recent Visual Studio *does* provide `vsnprintf` so it might be more preferable to get rid of this special case altogether)